docs/content/1.getting-started/4.icons/2.vue.md

```diff
@@ -32,6 +32,10 @@ props:
 You can use any name from the <https://icones.js.org> collection.
 ::
 
+::warning
+You have to separate the icon name from the collection with a colon (`:`). Learn more about the [Iconify naming convention](https://iconify.design/docs/icon-components/vue/#icon).
```

Comment:
```suggestion
When using collections with a dash (`-`), you need to separate the icon name from the collection name with a colon (`:`) as `@iconify/vue` does not handle this case like `@nuxt/icon`. For example, instead of `i-simple-icons-github` you need to write `i-simple-icons:github` or `simple-icons:github`.

Learn more about the [Iconify naming convention](https://iconify.design/docs/icon-components/vue/#icon).
```